File: exercism/python/grains/grains.py
"""
Module docstring
"""
def square(number):
    """
    :param number
    """
    if number not in range(1,65):
        raise ValueError("square must be between 1 and 64")
    return 2 ** (number -1)


def total():
    """
    Return the total number of grains on the board
    """
    return 2 ** 64 -1

File: exercism/kotlin/grains/src/main/kotlin/Grains.kt
import java.math.BigInteger

object Board {

    fun getGrainCountForSquare(number: Int): BigInteger {
        if(number < 1 || number > 64){
            throw IllegalArgumentException("square must be between 1 and 64");
        }
        return BigInteger.TWO.pow(number -1);
    }

    fun getTotalGrainCount(): BigInteger {
        return BigInteger.TWO.pow(64).subtract(BigInteger.ONE);
    }
}
